x-pack/plugins/osquery/package.json

```
@@ -16,6 +16,7 @@
     "nyc": "../../../node_modules/.bin/nyc report --reporter=text-summary",
     "junit:merge": "../../../node_modules/.bin/mochawesome-merge ../../../target/kibana-osquery/cypress/results/mochawesome*.json > ../../../target/kibana-osquery/cypress/results/output.json && ../../../node_modules/.bin/marge ../../../target/kibana-osquery/cypress/results/output.json --reportDir ../../../target/kibana-osquery/cypress/results && yarn junit:transform && mkdir -p ../../../target/junit && cp ../../../target/kibana-osquery/cypress/results/*.xml ../../../target/junit/",
     "junit:transform": "node ../security_solution/scripts/junit_transformer --pathPattern '../../../target/kibana-osquery/cypress/results/*.xml' --rootDirectory ../../../ --reportName 'Osquery Cypress' --writeInPlace",
+    "openapi:lint": "npx @redocly/cli lint --config ../../../packages/kbn-openapi-generator/redocly_linter/config.yaml $(find ./ -name '*.schema.yaml')",
```

Instead of using `npx`, call the installed CLI directly:

```suggestion
    "openapi:lint": "../../../node_modules/.bin/redocly lint --config ../../../packages/kbn-openapi-generator/redocly_linter/config.yaml $(find ./ -name '*.schema.yaml')",
```